_component_design/button-drop-down.md

```diff
@@ -0,0 +1,260 @@
+---
+title: Drop-Down Buttons
+parent: Inputs and Controls
+layout: component
+category: Components
+usage: >
+  A drop-down button is an element that reveals additional content when clicked by a user. There are two types of drop-down buttons: a basic drop-down button and a split drop-down button.
```

Comment:
first sentence: "A drop-down button is an element that, when clicked, reveals additional options to launch a process or initiate an action."